### ISSUE TITLE
Backup should be exposed only after the first payment 

### ISSUE THREAD

**Issue by kingonly** [breez/c-breez#395]
Like in Breez Mobile via a prompt and then a warning icon.

**Comment by roeierez** [comment]
We didn't implement backup in c-breez. For the alpha there is only mnemonics.

**Comment by kingonly** [comment]
I know. I was referring to the mnemonics. The flow should behave like the Breez Mobile backup though.
Also, we should display the mnemonics in the Security & Backup screen.

**Comment by erdemyerebasmaz** [comment]
> I know. I was referring to the mnemonics. The flow should behave like the Breez Mobile backup though.
> Also, we should display the mnemonics in the Security & Backup screen.

Progress so far:
- Mnemonics are now saved in the Keychain with #464 and verification of mnemonics is deferred and no longer part of registering a new node,
- Verification flow is moved to Security screen but verifying doesn't do anything at the moment

@kingonly Two questions.
- Do we want mnemonic phrase to be displayed on demand in the Security screen outside of verification flow?
- What is the difference between an account that hasn't verified their mnemonics v.s. one that has?
  - Will this only be used for Backup? If so I think we should implement Backup before going forward with this issue.

**Comment by kingonly** [comment]
1. Yes
2. Warning goes away if verified. Not sure I understand your comment about implementing backup. On Breez Cloud backup means saving the recovery phrase (let's use 'Recovery Phrase (Seed)' in the UI).
